src/engine/strat_engine/cmd.rs

```diff
@@ -67,8 +69,13 @@ pub fn verify_binaries() -> StratisResult<()> {
     match BINARIES.iter().find(|&(_, ref path)| path.is_none()) {
         None => Ok(()),
         Some((ref name, _)) => Err(StratisError::Error(format!(
-            "Unable to find absolute path for \"{}\"",
-            name
+            "Unable to find executable \"{}\" in any of {}. Did you install it?",
```

Comment:
Please give the bare information here, without the follow-up question or suggestion. In future, this could end up in several possible contexts, like a log message, or some further larger message, where the suggestion would be out of place.

Reply:
Ok, I'll remove the second part.